tests/Model/PackageTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace BringYourOwnIdeas\Maintenance\Tests\Model;
+
+use Package;
+use SapphireTest;
+
+class PackageTest extends SapphireTest
+{
+    public function testTitleFormatsNameCorrectly()
+    {
+        $nameTests = [
+            'pretendvendor/silverstripe-prefixedpackage' => 'prefixedpackage',
+            'pretend-vendor/silverstripe-hyphen-package' => 'hyphen-package',
+            'vendor/package' => 'package',
+            'vendor/hyphenated-package' => 'hyphenated-package',
+            'silverstripe/silverstripe-silverstripe-thing' => 'silverstripe-thing'
```

Comment:
Can you add a silverstripe-themes vendor example too? Also this is a perfect scenario to use a data provider :-)